libs/financial-aid/shared/src/lib/interfaces.ts

```diff
@@ -243,6 +244,16 @@ export interface PersonalTaxReturn {
   size: number
 }
 
+export interface DirectTaxPayment {
+  totalSalary: number
+  payerNationalId: string
+  personalAllowance: number
+  withheldAtSource: number
+  month: number
+  year: number
+  userType?: UserType
```

Comment:
Does a DirectTaxPayment not always have an userType?

Reply:
Nope it does not because I am reusing the interfaces for Ósk and for Veita.

In Ósk we don't have a user type but once we have written to our db we have it.

Reply:
Nope it does not because I am reusing the interfaces for Ósk and for Veita.

In Ósk we don't have a user type but once we have written to our db we have it.